encoders/numeric/FastICAEncoder/tests/test_fasticaencoder.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import pickle
+
+
+from .. import FastICAEncoder
+from encoders.numeric.test import TestClassNumeric
```

Comment:
we do not want to have these kind of dependencies, every test needs to be self contained, so every test will have to implement its tests (test_encode, test_load ....)